packages/cli/src/ceramic-daemon.ts

```diff
@@ -705,15 +721,25 @@ export class CeramicDaemon {
       }
     } else if (!parsedJWS.code) {
       return { error: 'Admin code is missing from the the jws block' }
-    } else if (shouldContainModels && (!parsedJWS.models || parsedJWS.models.length === 0)) {
+    } else if (
+      shouldContainModels &&
+      parsedJWS.models.length == 0 &&
+      parsedJWS.indices.length == 0
+    ) {
       return {
-        error: `The 'models' parameter is required and it has to be an array containing at least one model stream id`,
+        error: `At least one model must be specified, either in 'models' as a 'StreamID' or 'indices' as a 'ModelFieldIndex'`,
       }
     } else {
       return {
         kid: parsedJWS.kid,
         code: parsedJWS.code,
-        models: parsedJWS.models?.map((modelIDString) => StreamID.fromString(modelIDString)),
+        models: parsedJWS.models.map((modelIDString) => StreamID.fromString(modelIDString)),
+        indices: parsedJWS.indices.map((idx) => {
```

Comment:
I'm okay with duplicating the model streamids in the input and output of these commands in the short term to ease backwards compatibility.  But I'd like us to think about what the path would be to eventually condensing down to a format without that duplication.  If we eventually dropped the `models` field entirely, just having a field called `indices` would be odd.  Maybe we could call this `indexedModels`, `modelsInfo`, `modelsData` or some other similarly general-purpose name, and mark everywhere that `models` is deprecated in favor of the new field?

I also considered proposing the name `modelsWithIndices` but then if we ever add more information to the output format we'd want to rename it again.  Having the output format be an object now means it should be a bit more future proof as we can add new fields to the object in the future without breaking backwards compatibility.